queries/html/textobjects.scm

```diff
@@ -0,0 +1,22 @@
+(element) @function.outer
+
+(element (start_tag) . (_) @function.inner . (end_tag))
+
+((element (start_tag) . (_) @_start (_) @_end . (end_tag))
+ (#make-range! "function.inner" @_start @_end))
+
+(script_element) @function.outer
+(script_element (start_tag) . (_) @function.inner . (end_tag))
```

Comment:
duplication is needed to match a tag with only one element inside and with two or more.

Reply:
I meant to reference the lines 3-6, sorry I'm a little drunk...